nifi-extension-bundles/nifi-websocket-bundle/nifi-websocket-services-jetty/src/main/java/org/apache/nifi/websocket/jetty/JettyWebSocketServer.java

```diff
@@ -254,6 +261,21 @@ public Object createWebSocket(JettyServerUpgradeRequest servletUpgradeRequest, J
         }
     }
 
+    private static int getPort(JettyServerUpgradeRequest servletUpgradeRequest) {
+        final Object localSocketAddress = servletUpgradeRequest.getLocalSocketAddress();
+        if (localSocketAddress == null) {
+            throw new RuntimeException("Local socket address is null");
```

Comment:
This seems better reported as an `IllegalStateException`:
```suggestion
            throw new IllegalStateException("Local socket address is null");
```